src/pages/Login/Login.module.css

```
@@ -117,6 +117,48 @@
+    background-size: 0.75em;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.showPasswordRow input[type="checkbox"]:focus-visible {
+    outline: 2px solid #127BBE;
+    outline-offset: 2px;
+}
+
+.showPasswordLabel {
+    font-family: 'Merriweather', serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Merriweather" (font-family-name-quotes)

(font-family-name-quotes)